manuscripts/metrics/gerrit.py

```diff
@@ -62,6 +65,8 @@ def get_section_metrics(cls):
 
 
 class GerritMetrics(Metrics):
+    """ This class is the root one for all Gerrit metrics """
```

Comment:
This doc string is only in `gerrit` and `git`. Should we put this in all the data source files?